docs/UserGuide.md

```diff
@@ -6,24 +6,48 @@
+
+This app is perfect for you if you are a **Computer Science undergraduate student**
+
+This user guide is primarily designed for Computer Science undergraduates currently engaged in the internship finding process. Whether you are seeking you first internship or have prior experience, this guide is tailored to cater to your specific needs.
+
+### Prerequisite knowledge
+
+While this guide is accessible to a broad audience, we assume that you have a basic understanding of the internship-finding process. If you're new to this journey, don't worry! We've got you covered. Feel free to check out these [helpful resources](https://www.techinterviewhandbook.org/software-engineering-interview-guide/) or refer to our [glossary](#glossary) page for additional information to get you up to speed.
+
+Our goal is to make your internship application experience as smooth and successful as possible, regardless of your level of experience. Let's embark on this journey together and make your internship goals a reality.
+
+### Main features
+
+LetsGetHired allows you to add and organise your internship applications. View all applications at a glance, or narrow down your search with filter and search features. LetsGetHired also allows you to attach customized tags and notes, to track any additional information.
+
 <!-- * Table of Contents -->
 <page-nav-print />
 
 --------------------------------------------------------------------------------------------------------------------
 
 ## **Quick start**
 
-1. Ensure you have Java `11` or above installed in your Computer.
+1. Ensure you have Java `11` or above installed in your Computer (from [here](https://www.oracle.com/sg/java/technologies/javase/jdk11-archive-downloads.html) if you do not have it already).
 
 1. Download the latest `letsgethired.jar` from [here](https://github.com/AY2324S1-CS2103T-W17-2/tp/releases).
 
+    ![Zip file download](images/downloadJar.jpg)
+
 1. Copy the file to the folder you want to use as the _home folder_ for your **LetsGetHired** application.
 
 1. Open a command terminal, `cd` into the folder you put the jar file in, and use the `java -jar letsgethired.jar` command to run the application. A GUI similar to the below should appear in a few seconds. Note how the app contains some sample data.<br>
-   ![Ui](images/Ui.png)
+   
+    ![Ui](images/Ui.jpg)
+
+1. Here's what you'll see when you start the app for the first time ! We've added some intern applications already. Enter your commands in the **Command Bar**. On the left you'll see a **List View** of all your applications ordered by the date you added them. On the right, the **Select View** shows you a more in depth view of your selected application
 
 1. Type the command in the command box and press `Enter` to execute it. e.g. typing **`help`** and pressing Enter will open the help window. Some example commands you can try:
```

Comment:
Perhaps we should change the sample commands here?

Maybe we could include:
* `list`
* `view 1` 

Also `tag` feature is outdated - we have updated changing Status using `edit` instead!